tests/test_errors.py

```diff
@@ -28,6 +28,7 @@
     A,
     Color,
     ConcretePlugin,
+    Config,
```

Comment:
Please don't use Config as a test class :)
This is a great way to ensure confusion.